### ISSUE TITLE
Radiobeacon Android: Unable to upload database

### ISSUE THREAD

**Issue by timofonic** [wish7code/openbmap#100]
Hello.

I have tons of scanned BTPs but I'm unable to upload databases. I use Android 4.4.4 under ZTE A450.

What am I doing wrong? So I need to sign up?

Kind regards.

**Comment by wish7code** [comment]
Any news? Feel free to reopen with more details..

**Comment by timofonic** [comment]
Hello.

Do I write here or in another issue? Are there a mailing list?

I got "bored" of most GPS stuff because found it unreliable most of the time. I suspect the GPS reception of my mobile is crappy too (I got a Beaglebone Black, would be possible to use a very hq reception GPS+4G/LTE+Wifi  module with this or similar hardware in a portable way?). I used both Google Maps and OSMAnd maps, the experience was quite subpar.

I hoped Transportr would support more public transport services all over Europe, but no way. I tried to help, but they asked me certain data about databases and I got very busy IRL.

I would like to try it again. One of the things is that I find quite cumbersome to use a few apps for GPS "geocaching" "geomapping" and even contribute to different databases. I have severe spatial and time management perception issues too.
